doc/sphinx/source/introduction.rst

```diff
@@ -50,6 +50,80 @@ these challenging objectives the ESMValTool is conceived as a framework
 which allows community contributions to be bound into a coherent
 framework.
 
+.. _contact:
+
+Contact
+=======
+
+See `www.esmvaltool.org <https://www.esmvaltool.org>`_ for general contact information.
```

Comment:
This link is a bit confusing. Is this referring to the contact emails in the footer?

Reply:
Yes

Reply:
Do you have a suggestion for how this could be made less confusing?

Reply:
Perhaps this is something we should change on the website end instead of here.